### ISSUE TITLE
`S` does not focus search on /explore

### ISSUE THREAD

**Issue by hugurp** [internet4000/radio4000#312]
It already works on `/channel/tracks`, we should harmonize that and then show the keybindings in both inputs placeholders.

Related bonus: #265

**Comment by oskarrough** [comment]
Yes! Have to remember to use <kbd>shift+s</kbd> to avoid conflict with shuffle.
